.github/workflows/publish-release.yml

```
@@ -71,6 +71,17 @@ jobs:
         echo "::set-output name=prerelease::$prerelease"
       shell: bash
 
+    - name: Set up S3cmd and configure AWS credentials
+      uses: s3-actions/s3cmd@v1.1
+      with:
+        provider: aws
+        region: "us-east-1"
+        access_key: ${{ secrets.PREBUILDS_S3_ACCESS_KEY }}
+        secret_key: ${{ secrets.PREBUILDS_S3_SECRET_KEY }}
+
+    - name: Upload prebuilds to S3
+      run: s3cmd put --recursive --acl-public prebuilds/realm-* s3://${{ secrets.PREBUILDS_S3_BUCKET_NAME }}/realm-js-prebuilds/${{ steps.get-version.outputs.version }}
```

With [configure-aws-credentials](https://github.com/aws-actions/configure-aws-credentials) this will be
```suggestion
      run: aws s3 sync prebuilds s3://${{ secrets.PREBUILDS_S3_BUCKET_NAME }}/${{ steps.get-version.outputs.version }}
```